pkgpanda/build/__init__.py

```diff
@@ -773,6 +774,10 @@ def cache_abs(filename):
 
     final_buildinfo['sources'] = sources
 
+    if builder.has('sysctl'):
```

Comment:
In this case should just be `builder.take('sysctl')`. load_buildinfo() filling in a default if none is given mean sit is always set.